Objects/methodobject.c

```diff
@@ -251,13 +251,9 @@ static Py_hash_t
 meth_hash(PyCFunctionObject *a)
 {
     Py_hash_t x, y;
-    if (a->m_self == NULL)
-        x = 0;
-    else {
-        x = PyObject_Hash(a->m_self);
-        if (x == -1)
-            return -1;
-    }
+    x = _Py_HashPointer(a->m_self);
+    if (x == -1)
```

Comment:
Does this need braces according to PEP7?

Reply:
Right. Although this is an old code, just unindented.

But actually this check is not needed because `_Py_HashPointer()` never returns -1.